sdk/OoyalaSkinSDK/widgets/ResponsiveList.js

```diff
@@ -67,20 +67,25 @@ var ResponsiveList = React.createClass({
 
   renderSlice: function(slice: object, i: number) {
     var sliceStyle = this.props.horizontal ? styles.column : styles.row;
+
+    renderItem = this.renderItem;
```

Comment:
is it possible to add the 'var' to the front of it?  it feels like this is leaking a little out of scope